libqtile/scripts/run_cmd.py

```diff
@@ -81,7 +81,7 @@ def add_subcommand(subparsers, parents):
         help="Do not break on match (keep applying rules).",
     )
     parser.add_argument("-g", "--group", help="Set the window group.")
-    parser.add_argument("cmd", help="Command to execute."),
+    (parser.add_argument("cmd", help="Command to execute."),)
```

Comment:
this should be
```suggestion
    parser.add_argument("cmd", help="Command to execute."
```
right?